### DISCUSSION TITLE
treeTable.reloadData无法设置cols

### DISCUSSION THREAD

**Discussion by hjingsuper** [layui/layui#1886] [讨论]
![image](https://github.com/layui/layui/assets/36070786/fce415c2-6f44-4e53-9ec7-c94e5c91df78)
看文档说是table.reloadData与treeTable.reloadData一致，前者可以重新设置cols，后者则不行
示例：[https://codepen.io/hjingsuper/pen/BaeaxxO](https://codepen.io/hjingsuper/pen/BaeaxxO)

**Comment by Sight-wcg** [comment]
`table.reloadData` 和 `treeTable.reloadData` 均不支持重新设置 cols，不支持的参数见
https://github.com/layui/layui/blob/7f610ba4458d980b927f4be2308a0d9ae5b0b1ce/src/modules/table.js#L3120-L3136

**Reply by hjingsuper** [reply]
明白
